src/components/Player/Pages/Overview/Summary.jsx

```diff
@@ -34,19 +35,28 @@ const SummOfRecMatches = ({ matchesData }) => {
 
   const dataKeys = Object.keys(data);
 
-  const numRows = Math.min(MAX_MATCHES_ROWS, matchesData.length);
+  const desiredNumRows = Math.min(MAX_MATCHES_ROWS, matchesData.length);
 
-  for (let i = 0; i < numRows; i += 1) {
+  // XXX - getPlayerRecentMatches seems to only return a maximum of 20 matches
+  // meaning that this set of "validMatches" can be fewer than 20. Tried
+  // switching to the getPlayerMatches endpoint but that does not provide
+  // data on several fields (e.g. gold_per_min, xp_per_min, last_hits, etc.)
+  const validMatches = _.take(_.filter(matchesData, (match) => {
```

Comment:
I think we are trying to avoid lodash and others, so u can do it by
```js
const validMatches = matchesData
    .filter(match => match.game_mode !== 19)
    .slice(0, desiredNumRows);
```